ui/components/multichain/token-list-item/token-list-item.tsx

```diff
@@ -396,72 +422,20 @@ export const TokenListItem = ({
                   if (isFullScreen) {
                     history.push(NETWORKS_ROUTE);
                   } else {
-                    global.platform.openExtensionInBrowser(NETWORKS_ROUTE);
+                    global.platform &&
+                      typeof global.platform.openExtensionInBrowser ===
+                        'function' &&
+                      global.platform.openExtensionInBrowser(NETWORKS_ROUTE);
```

Comment:
any reason this wouldn't work?

```suggestion
                    global.platform.openExtensionInBrowser?.(NETWORKS_ROUTE);
```

Reply:
huh, that seems to work fine actually. I thought I tried that but failed. I'll commit your change ty 👍